pkg/apis/extensions/validation/validation_test.go

```diff
@@ -2813,3 +2820,51 @@ func TestIsValidSysctlPattern(t *testing.T) {
 		}
 	}
 }
+
+func Test_validatePSPRunAsUser(t *testing.T) {
+	var testCases = []struct {
+		runAsUserStrategy extensions.RunAsUserStrategyOptions
+		fail              bool
+	}{
+		{extensions.RunAsUserStrategyOptions{Rule: extensions.RunAsUserStrategy("someInvalidStrategy")}, true},
+		{extensions.RunAsUserStrategyOptions{Rule: extensions.RunAsUserStrategyMustRunAs}, false},
+		{extensions.RunAsUserStrategyOptions{Rule: extensions.RunAsUserStrategyRunAsAny}, false},
+		{extensions.RunAsUserStrategyOptions{Rule: extensions.RunAsUserStrategyMustRunAsNonRoot}, false},
+		{extensions.RunAsUserStrategyOptions{Rule: extensions.RunAsUserStrategyMustRunAsNonRoot, Ranges: []extensions.UserIDRange{{Min: 2, Max: 3}, {Min: 4, Max: 5}}}, false},
```

Comment:
seems like this one should fail because you can't use run as nonroot with a range - am I missing something?